scripts/memory_logger.py

```diff
@@ -783,7 +783,7 @@ def buildGraph(self):
         suffixes = ["Terabytes", "Gigabytes", "Megabytes", "Kilobytes", "Bytes"]
         multiplier = 1 << 40;
         index = 0
-        while largest_memory[-1] < multiplier and multiplier >= 1:
+        while largest_memory[-1] < multiplier and multiplier > 1:
```

Comment:
This is probably the only fix you need. This would ensure that you don't divide by zero, which was what the problem was.